### ISSUE TITLE
Performance is not getting better

### ISSUE THREAD

**Issue by henry090** [rstudio/keras#625]
Hi, I cannot get a satisfying accuracy from ["Naming and locating objects in images" post](https://blogs.rstudio.com/tensorflow/posts/2018-11-05-naming-locating-objects/). 
Validation accuracy is around 5%... But you could achieve better results.
[single-object-detection.log](https://github.com/rstudio/keras/files/2718030/single-object-detection.log)

**Comment by skeydan** [comment]
What is the question?

**Comment by skeydan** [comment]
Referring to the post in general, as well as to its successor on multiple object detection, I think that for generalization on this dataset (with 20 different classes and the dataset-inherent difficulties described in the posts), the most important thing would be using data augmentation.
Given the complexity involved already (especially in the second post), I decided to leave that to a later occasion and concentrate on the concepts/general techniques involved.
But it is certainly something to be pursued.
